instructions.md

```diff
@@ -0,0 +1,352 @@
+        --peak-tol <PEAK_TOL>
+            Absolute peak m/z tolerance
+
+            The error tolerance used to match database spectra peak m/zs to query spectra peaks.
+            Peaks in the spectral database that have a difference in m/z bigger than this are
+            considered to have distinct m/zs, otherwise they are considered to be (approximately)
+            equal. The indexing strategy relies on this - a smaller tolerance means we divide
+            spectra into more bins and this implies a more fine-grained comparison.
+
+            [default: 0.02]
+
+Spectra:
+        --min-peak-count <MIN_PEAK_COUNT>
+            Minimum number of spectral peaks
+
+            Any spectra with fewer peaks than this will be filtered out. This also serves as the
+            minimum number of peaks that can be returned after the peak merging operation.
+
+            [default: 1]
+
+        --no-filter
+            Disable peak filtering
+
+        --no-merge
+            Disable peak merging
+
+        --normalize
+            Enable peak intensity normalization
+
+        --peak-filter-window <PEAK_FILTER_WINDOW>
+            Window size for peak filtering (Da)
+
+            This value is used in the filtering pre-processing step of the spectrum. This removes
+            low intensity peaks by keeping only the --peaks-per-window most intense peaks in
+            non-overlapping m/z windows of size --peak-filter-window. Filtering pre-preprocessing
+            can be skipped with --no-filter.
+
+            [default: 50.0]
+
+        --peak-merge-thresh <PEAK_MERGE_THRESH>
+            Threshold used to merge peaks (Da)
+
+            Peaks with m/zs within this value of each other will be merged into a single peak with
+            intensity aggregated additively. The minimum number of peaks that can remain after the
+            merge pre-processing step is defined by --min-peak-count. Merging pre-preprocessing can
+            be skipped with --no-merge.
+
+            [default: 0.05]
+
+        --peaks-per-window <PEAKS_PER_WINDOW>
+            Number of peaks to keep per window
+
+            This value is used in the filtering pre-processing step of the spectrum. This removes
+            low intensity peaks by keeping only the --peaks-per-window most intense peaks in
+            non-overlapping m/z windows of size --peak-filter-window. Filtering pre-preprocessing
+            can be skipped with --no-filter.
+
+            [default: 5]
+
+    -s, --spectra-files <PATHS>...
+            Paths to files containing the spectra of interest
+
+            To use more than one spectrum add multiple, space-separated files.
+```
+    
+</details>
+
+### Examples
+
+```sh
+./masst_plus exact index -d path/to/exact_index1 -s path/to/one_million_50.mgf
+```
+This creates an exact index for the spectra in `one_million_50.mgf` and adds it to the index at `path/to/exact_index1`, with the default options for spectrum preprocessing and filtering.
```

Comment:
does this add to the index at -d or replace the index at -d?

Reply:
It will result in an error if there is already a index at -d

Reply:
I'll add this to the docs